kapeta.yml

```diff
@@ -0,0 +1,72 @@
+kind: core/resource-type-operator
+metadata:
+  name: kapeta/resource-type-npm-repository
+  title: NPM Repository
+  description: Provides a npm repository resource to your block
+  visibility: public
+spec:
+  icon:
+    type: fontawesome5
+    value: fab fa-npm
+  ports:
+    - type: http
+  local:
+    image: verdaccio/verdaccio:5.29.2
+    ports:
+      http:
+        port: 4873
+        type: tcp
+    mounts:
+      conf: /verdaccio/conf
+      storage: /verdaccio/storage
+      plugins: /verdaccio/plugins
+    credentials:
+      username: admin
+      password: admin
+---
+kind: core/resource-type-operator
+metadata:
+  name: kapeta/resource-type-docker-repository
+  title: Docker Repository
+  description: Provides a docker repository resource to your block
+  visibility: public
+spec:
+  icon:
+    type: fontawesome5
+    value: fab fa-docker
+  ports:
+    - type: http
+  local:
+    image: registry:2.8.3
+    ports:
+      http:
+        port: 5000
+        type: tcp
+    mounts:
+      data: /var/lib/registry
+---
+kind: core/resource-type-operator
+metadata:
+  name: kapeta/resource-type-maven-repository
+  title: Maven Repository
+  description: Provides a maven repository resource to your block
+  visibility: public
+spec:
+  icon:
+    type: url
+    value: https://storage.googleapis.com/kapeta-public-cdn/icons/maven.svg
+  ports:
+    - type: http
+  local:
+    image: dzikoysk/reposilite:3.4.6
```

Comment:
perhaps we should mirrors these images??

Reply:
We dont really elsewhere though ? You're thinking so they dont suddenly disappear ?

Reply:
Exactly

Reply:
This live on dockerhub - so should be ok. But add a ticket for it ? It's a good idea - we then just need to be consistent imo